sdk/core/Azure.Core/README.md

```diff
@@ -82,9 +83,25 @@ SecretClient client = new SecretClient(new Uri("http://example.com"), new Defaul
 
 More on client configuration in [client configuration samples](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/Configuration.md)
 
+### Customzing Request Using `RequestContext`
+
+Besides general configuration of _serice client_ through `ClientOptions`, you can customize each individual request sent by _service client_
```

Comment:
```suggestion
Besides general configuration of _service clients_ through `ClientOptions`, it is possible to customize each individual request sent by a _service client_
```

Reply:
You can only do this in protocol methods on a service client, and in select APIs on Generation1 Convenience Clients, where RequestContext is exposed as a parameter.

I'd suggest changing this to "[...] you can customize the request send by the _service client_ using protocol methods or convenience APIs that expose `RequestContext` as a parameter."

Reply:
Done